doc/classes/Input.xml

```diff
@@ -360,6 +360,15 @@
 				[b]Note:[/b] This value can be immediately overwritten by the hardware sensor value on Android and iOS.
 			</description>
 		</method>
+		<method name="should_ignore_device" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="vendor_id" type="int" />
```

Comment:
```suggestion
			<argument index="0" name="vendor_id" type="int" />
```
See the surrounding code, you need to adapt the code to 3.x

Reply:
> See the surrounding code, you need to adapt the code to 3.x

Got it with 570b864daa7ba1f5ab9b54b7bb9445228ef0ab0f. Thank you.